### PR TITLE
done changes

### PR DESCRIPTION
**Pull request by vladp456**

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Default ignored files
```
Comment (pospielov) [inline .idea/.gitignore:1]: Please remove all files `./idea/*` from commit

File: functions/index.js [hunk 2]
```diff
@@ -51,12 +51,12 @@ const common_functions = {
...
-     * @param {String} path 
-     * @returns 
+     * @param {String} path
+     * @returns
      */
     isPathRelative(path) {
-        if(typeof path !== 'string') return false 
-        let isAbsolute = /^(?:\/|[a-z]+:\/\/)/.test(path);
+        if(typeof path !== 'string') return false
+        let isAbsolute = /^([a-zA-Z]\:|\\\\[^\/\\:*?"<>|]+\\[^\/\\:*?"<>|]+)(\\[^\/\\:*?"<>|]+)+(\.[^\/\\:*?"<>|]+)$/.test(path);
```
Comment (pospielov) [inline functions/index.js:59]: Do we need this change even though we changed the order of `if ... else...` in another file?

**Review by pospielov — CHANGES_REQUESTED** [review]
Why do we need `__tests__/isBase64.test.js` file? If you didn't set up the test, just delete it. If you set up the test, could you make this file smaller? Like 30kb instead of 1Mb?

File: functions/index.js (hunk 2 above)
Comment (vladp456) [reply functions/index.js:59]: Yes, we need it.

File: .idea/.gitignore (hunk 1 above)
Comment (vladp456) [reply .idea/.gitignore:1]: We need __tests__/isBase64.test.js as well as we need other tests such as isUrl or isPath. I set up it but I used an image as a base64 data that is why it is 1Mb. I will try to use another small type of base64 data to solve that problem.

**Review by pospielov — APPROVED** [review]

**Comment by vladp456** [comment]
I have read the CLA Document and I hereby sign the CLA

**Pull request merged**